examples/nerdctl-ipfs-registry-kubernetes/README.md

```diff
@@ -1,95 +1,40 @@
-$ kind create cluster --image=kindest/node:v1.23.1 --config=/tmp/kindconfig.yaml
-$ ./bootstrap.yaml.sh > ./bootstrap.yaml
-$ kubectl apply -f .
-```
-
-Prepare `kind-worker` (1st node) for importing an image to IPFS
-
-(in `kind-worker`)
-
-```console
-$ docker exec -it kind-worker /bin/bash
-(kind-worker)# NERDCTL_VERSION=0.15.0
-(kind-worker)# curl -sSL --output /tmp/nerdctl.tgz https://github.com/containerd/nerdctl/releases/download/v${NERDCTL_VERSION}/nerdctl-${NERDCTL_VERSION}-linux-amd64.tar.gz
-(kind-worker)# tar zxvf /tmp/nerdctl.tgz -C /usr/local/bin/
-```
-
-Add an image to `kind-worker`.
-
-```console
-$ docker exec -it kind-worker /bin/bash
-(kind-worker)# mkdir -p /tmp/ipfsapi ; echo -n /ip4/127.0.0.1/tcp/5001 >  /tmp/ipfsapi/api
-(kind-worker)# export IPFS_PATH=/tmp/ipfsapi
-(kind-worker)# nerdctl pull ghcr.io/stargz-containers/jenkins:2.60.3-org
-(kind-worker)# nerdctl push ipfs://ghcr.io/stargz-containers/jenkins:2.60.3-org
-(kind-worker)# nerdctl rmi ghcr.io/stargz-containers/jenkins:2.60.3-org
-```
-
-The image added to `kind-worker` is shared to `kind-worker2` via IPFS.
-You can run this image on all worker nodes using the following manifest (assuming the image is added to IPFS as CID `localhost:5050/ipfs/bafkreife3j4tgtx23jcautprornrdp2p4g3j3ndnidzdlrpd7unbpnwkce`).
-
-```console
-$ cat <<EOF | kubectl apply -f -
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: jenkins
-spec:
-  replicas: 2
-  selector:
-    matchLabels:
-      app: jenkins
-  template:
-    metadata:
-      labels:
-        app: jenkins
-    spec:
-      containers:
-      - name: jenkins
-        image: localhost:5050/ipfs/bafkreife3j4tgtx23jcautprornrdp2p4g3j3ndnidzdlrpd7unbpnwkce
-        resources:
-          requests:
-            cpu: 1
-EOF
-```
+The above examples use `nerdctl ipfs regisry` running in a Pod.
+The image is available at [`ghcr.io/stargz-containers/nerdctl-ipfs-registry`](https://github.com/orgs/stargz-containers/packages/container/package/nerdctl-ipfs-registry).
 
-The image runs on all nodes.
+The following Dockerfile can be used to build it by yourself.
 
-```console
-$ kubectl get pods -owide | grep jenkins
-jenkins-7bd8f96d79-2jbc6          1/1     Running   0          69s    10.244.1.3   kind-worker    <none>           <none>
-jenkins-7bd8f96d79-jb5lm          1/1     Running   0          69s    10.244.2.4   kind-worker2   <none>           <none>
-```
+```Dockerfile
+FROM ubuntu:22.04 AS dev
 
-## Example Dockerfile of nerdctl
+ARG NERDCTL_VERSION=0.23.0
+ARG NERDCTL_AMD64_SHA256SUM=aa00cd197de3549469e9c62753798979203dc0607f3e60f119ed632478244553
+ARG NERDCTL_ARM64_SHA256SUM=bc8095b8d60a2f25da7e5c456705dce2db020a0a87d003093550994618189ea3
+ARG NERDCTL_PPC64LE_SHA256SUM=162d68a636e0a9c32f705f27390ae8ed919ca8c0442832909ebf3c0e5a884fac
+ARG NERDCTL_RISCV64_SHA256SUM=1580fe87e730fe4b4442ce3e5199fa487ca03dcd0761f0bfa3c7603e4be10372
+ARG NERDCTL_S390X_SHA256SUM=7905ef258968c6f331944a097afe28251c793471fdbc4b7e87aae63f999e8098
```

Comment:
I'd rather prefer to just set `ARG NERDCTL_SHA256SUMS_SHA256SUM` (sha256sum of the `SHA256SUMS` file), but it can be another PR